app/main/routes.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+from app import mongo
+from flask import Blueprint, current_app, render_template
+
+main = Blueprint("main", __name__)
+
+# database collection variables
+desserts_collection = mongo.db.desserts
+recipes_collection = mongo.db.recipes
+
+#----- Global Helper -----#
+def get_total_recipes():
+        return int(recipes_collection.count())
+@main.context_processor
+def total_recipes():
+        return dict(total_recipes=get_total_recipes)
+
+
+#----- HOME -----#
+@main.route("/")
+def home():
+        carousel = recipes_collection.aggregate([{"$sample": {"size": 8}}])

```

Comment:
You seem to be using 8 spaces/ 2 tabs indentation, that's much. Use 4 instead.

Reply:
This is coming from VS Code automatically. I'm not manually typing any spaces or tabs. If I start to build a function, as soon as I hit <enter> after **def home():** then it automatically breaks to the next line indented that far... I figured it was common practice, since VS Code is doing it automatically? Let me know if I need to manually decrease the tabbing so it's 4 chars only?